config/application.rb

```diff
@@ -58,7 +58,7 @@ class Application < Rails::Application
     # This will create an empty whitelist of attributes available for mass-assignment for all models
     # in your app. As such, your models will need to explicitly whitelist or blacklist accessible
     # parameters by using an attr_accessible or attr_protected declaration.
-    config.active_record.whitelist_attributes = true
+    # config.active_record.whitelist_attributes = true
```

Comment:
Maybe add a comment here about why you commented this out?

Reply:
I removed that line and the comment above it in the latest commit. When uncommented, it would cause this warning:

"DEPRECATION WARNING: config.active_record.whitelist_attributes is deprecated and have [sic] no effect. Remove its call from the configuration."

We originally left it in because we weren't 100% convinced that is true, but at this point I think it's safe to remove.